src/Php/PhpPrinterIndentationDetectorVisitor.php

```diff
@@ -39,7 +39,7 @@ public function enterNode(Node $node)
 			return null;
 		}
 
-		if (count($node->stmts) === 0) {
+		if ($node->stmts === null || count($node->stmts) === 0) {
```

Comment:
I wonder why phpstan did not error regarding count(mixed)

Reply:
I don't use level 9 (but maybe I should for this small library).

Reply:
I've tried running PHPStan on level 9 here, and the results were the same before and after the patch: total 5 errors, none related to this `count()`.